src/components/claims/claim-details/index.tsx

```diff
@@ -150,21 +156,17 @@ const ClaimPage: React.SFC<Props> = ({ match }) => (
             />
           </Grid>
           <Grid item xs={12}>
-            <>
-              <Grid item xs={12}>
-                <FileUpload
-                  claimId={match.params.id}
-                  memberId={member.memberId}
-                  onUploaded={() => refetch()}
-                />
-              </Grid>
-              <Grid item xs={12}>
-                <ClaimFileTable
-                  claimFiles={claimFiles}
-                  claimId={match.params.id}
-                />
-              </Grid>
-            </>
+            <CreateTicketStandAlone claimId={match.params.id} />
```

Comment:
This could probably be named referenceId instead
```suggestion
            <CreateTicketStandAlone referenceId={match.params.id} />
```